kubernetes/utils/delete_from_yaml.py

```diff
@@ -0,0 +1,147 @@
+import re
+from os import path
+
+import yaml
+
+from kubernetes import client
+
+
+def delete_from_yaml(
```

Comment:
overall, what is the difference between this and create_from_yaml? could you please list the differences in the PR description? is it possible to reuse to avoid dups?

Reply:
Yes I will do it

Reply:
I have written the difference between the methods create_from_yaml and delete_from_yaml editing the PR description and also fixed the issues as stated earlier.